plugin.audio.talandaviad/addon.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.talandaviad" name="Tal &amp; Aviad" version="0.0.1" provider-name="ECO 99FM">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0"/>
```

Comment:
2.25.0 to match krypton api version please. Reasoning is here: http://kodi.wiki/view/addon.xml